session/session.go

```diff
@@ -312,32 +313,32 @@ func sanitizeBalInfo(balInfo perun.BalInfo) {
 	}
 }
 
-// retrieveParts retrieves the peers from corresponding to the aliases from the contacts provider.
+// retrieveParts retrieves the peer IDs corresponding to the aliases from the ID provider.
```

Comment:
```suggestion
// retrievePartIDs retrieves the peer IDs corresponding to the aliases from the ID provider.
```

See the comment on the function definition (below).